manifests/v2/third-party/jobset/jobset_manager_config.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: config.jobset.x-k8s.io/v1alpha1
+kind: Configuration
+namespace: kubeflow-system
```

Comment:
You don't need this.

https://github.com/kubernetes-sigs/jobset/blob/release-7.1/api/config/v1alpha1/defaults.go#L44

If jobset is not configured with a namespace option, it will take the namespace from the serviceaccount.

Reply:
Oh, good point.
Even without my PR, it will take namespace from the SA.